src/contexts/Api/index.tsx

```diff
@@ -66,36 +71,14 @@ export const APIProvider = ({ children }: { children: React.ReactNode }) => {
     !!localStorage.getItem('isLightClient')
   );
 
-  // initial connection
   useEffect(() => {
-    if (!provider) {
-      const _network: NetworkName = localStorage.getItem(
-        'network'
-      ) as NetworkName;
```

Comment:
This is unncesessary any more, since from now on [the `network` state](https://github.com/Cardinal-Cryptography/aleph-zero-dashboard/pull/37/files#diff-de701a1090eab6a6ce998b10578ba8d16bd3d81a57a7caf3ea8d1d14141ab0a8R48) is our single source of truth and it's always set.